plugins/azure-sites/src/components/AzureSitesOverviewTableComponent/AzureSitesOverviewTable.test.tsx

```diff
@@ -65,7 +66,7 @@ describe('AzureSitesOverviewWidget', () => {
   });
 
   it('should display an overview table with the data from the requests', async () => {
-    const rendered = render(
+    const rendered = await renderInTestApp(
```

Comment:
looks like switching away from using `render` here [makes the import unused](https://github.com/backstage/backstage/actions/runs/4245218628/jobs/7380437688#step:7:10)

Reply:
Indeed, will fix.